src/core/unittest/FrameTest.cpp

```diff
@@ -243,6 +243,52 @@ TEST(FrameTest, ReliableResetStreamFrameEncodeDecode)
     ASSERT_EQ(Frame.ReliableSize, DecodedFrame.ReliableSize);
 }
 
+BOOLEAN DidHitReorderingThreshold(
```

Comment:
Thoughts on renaming this to simply `TestReorderingThreshold`? Also, let's add a comment to explain it some.